apps/web/components/Organization/Slice/TwoColumnText/TwoColumnTextSlice.tsx

```diff
@@ -44,7 +44,7 @@ export const TwoColumnTextSlice: React.FC<
             </GridColumn>
             <GridColumn span={columnSpan} hiddenBelow="lg">
               {slice.rightTitle && (
-                <Text variant="h2" as="h2">
+                <Text variant="h2" as="h2" id={labelId}>
```

Comment:
We've already got the label id on the leftTitle right? Do we want to have the id potentially on both elements?